draft-ietf-webtrans-http2.md

```diff
@@ -117,38 +118,48 @@ WebTransport servers are identified by an HTTPS URI as defined in {{Section
 4.2.2 of HTTP}}.
 
 When an HTTP/2 connection is established, both the client and server have to
-send a SETTINGS_ENABLE_WEBTRANSPORT setting in order to indicate that they
-both support WebTransport over HTTP/2.
+send a SETTINGS_ENABLE_WEBTRANSPORT setting to indicate that they both support
+WebTransport over HTTP/2.
+
+The server also needs to send a SETTINGS_WEBTRANSPORT_MAX_SESSIONS setting with
```

Comment:
It would be nice to not have both, but in practice we have the following setup: 
- Both sides send ENABLE_WEBTRANSPORT to say "I'm willing to talk WebTransport, this RFC is now in play"
- Only the server sends MAX_SESSIONS, set to a non-zero value, to say "I'm willing to accept this many sessions" 
The client cannot accept sessions, since the server cannot send CONNECT to the client.

If we wanted to have just the one setting, we could in theory have the client send it with a value of 0, but that's kinda messy for H2 settings, since existence of a setting that is set to 0 is not a great signal that things are set up to handle well.
We could also do silly things like having the value you accept be `<sent value> - 1`, but that's sadness.